iwyu_preprocessor.h

```diff
@@ -345,6 +356,15 @@ class IwyuPreprocessorInfo : public clang::PPCallbacks,
   // per file in the current inclusion chain..
   stack<clang::SourceLocation> begin_exports_location_stack_;
 
+  // For processing pragmas. It is the current stack of open
+  // "begin_keep".     There should be at most one item in this stack
```

Comment:
The two spaces after period for `begin_exports_location_stack_` is some kind of editor setting from the original authors, I think. You can use two spaces or one, but this alignment looks funny. I think the most reasonable formatting would be:
```
  // For processing pragmas. It is the current stack of open "begin_keeps".
  // There should be at most one item in this stack per file in the current
  // inclusion chain.
```